runtime/lua/vim/filetype.lua

```diff
@@ -49,8 +52,23 @@ local extension = {
   art = "art",
   asciidoc = "asciidoc",
   adoc = "asciidoc",
+  asa = function(path, bufnr)
+    if vim.g.filetype_asa then
+      return vim.g.filetype_asa
+    else
+      return "aspvbs"
+    end
+  end,
   ["asn1"] = "asn",
   asn = "asn",
+  asp = function(path, bufnr)
+    if vim.g.filetype_asp then
+      return vim.g.filetype_asp
+    elseif getline(bufnr, 1, 3):find("perlscript") then
```

Comment:
Empty if branch?

Reply:
I think I accidentally removed it. I will fix it